Anonymous labels should not be exported with `rgbasm -E`

Hello,

I'm trying to learn the ropes of rgbds and I encountered an issue regarding the "Anonymous Labels"
described here: https://rgbds.gbdev.io/docs/v0.6.1/rgbasm.5#Anonymous_labels

My Project has multiple asm files, which get compiled individualy and then linked together in the end. 

Everything is managed via a Makefile.

this is my `main.asm`
```asm
SECTION "main", ROM0[$150]

include "core/memory.inc"

cLCDC   equ  $ff40
cBGMAP0 equ  $9800 ;- 9bff   used when LCDC s bit $10 is turned off

main::
    ; Disable screen
    ld hl, cLCDC
    res 7, [hl]
    
    ; fill map 0 (32*32 Byte) with zeros
    fill_mem cBGMAP0, $0400, 0
    
    ; Enable screen
    ld hd, CLCDC
    set 7, [hl]

; main-loop
:   halt
    nop
    jr :- 
```

This is the `core/memory.inc` that `main.asm` includes.
```asm
/*==============================================================================
 ## fill_mem(n16 adress, n16 length, n8 fillbyte)
 Fill a number of bytes at address HL with the value stored in A
 ------------------------------------------------------------------------------
 ### Input:
   - adress:  memory adress from which to fill
   - length: number of bytes to fill
   - fillbyte: byte to fill the memory with

 ### Destroys:
   - AF
   - HL
   - BC
 ============================================================================*/
MACRO fill_mem
    ld hl, \1
    ld a, \3 
    ld bc, \2
    call _fill_BC_bytes_at_HL_with_A
ENDM
```

And this is the `core/memory.asm` that contains the actual function that gets executed when `fill_mem()` is called
```asm
SECTION "core.memory", ROM0

include "./utils.inc"

_fill_BC_bytes_at_HL_with_A::
    push hl
    push bc
    push af

:   check_BC_zero
    jp z, :+
        pop af
        push af

        ld [hli], a
        dec bc
    jp :-
    
:   pop af
    pop bc
    pop hl
    ret
```

Both `main.asm` and `core/memory.asm` get compiled individualy.
```bash
...
rgbasm -h -E -i./src -i./src/core -osrc/main.asm.obj src/main.asm
rgbasm -h -E -i./src -i./src/core -osrc/core/memory.asm.obj src/core/memory.asm
```

and then linked together
```bash
rgblink -t -oday001-clearing_the_screen.debug.gb -nday001-clearing_the_screen.sym ./src/card.asm.obj ./src/main.asm.obj ./src/core/memory.asm.obj
```

which then results in this lovely little error here:
```bash
error: "!0" both in ./src/core/memory.asm.obj from src/core/memory.asm(10) and in ./src/main.asm.obj from src/main.asm(17)
make: *** [Makefile:14: debug] Fehler 1
```

--- a/src/asm/symbol.c
+++ b/src/asm/symbol.c
@@ -41,7 +41,7 @@ static char savedTIME[256];
 static char savedDATE[256];
 static char savedTIMESTAMP_ISO8601_LOCAL[256];
 static char savedTIMESTAMP_ISO8601_UTC[256];
-static bool exportall;
+static bool exportAll;
 
 bool sym_IsPC(struct Symbol const *sym)
 {
@@ -513,7 +513,8 @@ static struct Symbol *addLabel(char const *symName)
 	// If the symbol already exists as a ref, just "take over" it
 	sym->type = SYM_LABEL;
 	sym->value = sect_GetSymbolOffset();
-	if (exportall)
+	// Don't export anonymous labels
+	if (exportAll && symName[0] != '!')
 		sym->isExported = true;
 	sym->section = sect_GetSymbolSection();
 
@@ -668,7 +669,7 @@ struct Symbol *sym_Ref(char const *symName)
 // Set whether to export all relocatable symbols by default
 void sym_SetExportAll(bool set)
 {
-	exportall = set;
+	exportAll = set;
 }
 
 static struct Symbol *createBuiltinSymbol(char const *symName)

